README.md

```diff
@@ -27,8 +27,7 @@
 Before you begin, you need to install the following tools:
 
 - [Node (v18 LTS)](https://nodejs.org/en/download/)
-- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
-- [Git](https://git-scm.com/downloads)
```

Comment:
Was removing `Git` an accident or you don't want it there? haha

Reply:
fixed at https://github.com/scaffold-eth/scaffold-eth-2/pull/444/commits/ec68f7075de0e07e2757e5556e24ff426ee079e7